config/jobs/build-drivers/build-new-almalinux.yaml

```diff
@@ -0,0 +1,97 @@
+presubmits:
+  falcosecurity/test-infra:
+  - name: validiate-new-drivers-debian-presubmit
```

Comment:
Name `debian` is wrong here :)

In whole file you need to substitute `debian` with `almalinux` .

Reply:
Oh wow, I could've sworn I did that, my bad!

Reply:
:D np!

Reply:
I pushed the changes now.